### DISCUSSION TITLE
How to couple two plugins in a single trial and get "continuous" data/responses from a slider ?

### DISCUSSION THREAD

**Discussion by FedericoCurzel** [jspsych/jsPsych#1392] [Q&A]
Hi everyone!

I am trying to setup an experiment and I am facing two problems:
1. In every trial I need to have two plugins: a **slider** and a **keyboard response**, together with a **video** and a _separated_ **audio** that will be my stimuli. How can I do it? ; 
2. I need to record "continuous" responses from the slider (get the slider position every 'X' seconds during the trial) and also to record every keyboard (ex. Space bar) touch. 

Every trial lasts 1 minute. 

Thanks in advance for your help! 

Federico Curzel

**Comment by jodeleeuw** [comment]
Moving this over to a discussion thread

**Comment by FedericoCurzel** [comment]
Hello! 

Today I could partially solve the issue. I will write here eventual related problems in the next days.  
Thanks for have noticed the issue!  

Federico Curzel
